partitioned-table.md

```diff
@@ -195,7 +195,13 @@ It will partition the data by year and by name in the ranges ['', 'G'), ['G', 'M
 
 ### Range INTERVAL partitioning
 
-Range INTERVAL partitioning is an extension of Range partitioning, which allows you to create partitions of a specified interval easily. Starting from v6.3.0, INTERVAL partitioning is introduced in TiDB as syntactic sugar:
+Range INTERVAL partitioning is an extension of Range partitioning, which allows you to create partitions of a specified interval easily. Starting from v6.3.0, INTERVAL partitioning is introduced in TiDB as syntactic sugar.
+
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use it in the production environment.
```

Comment:
```suggestion
> This is still an experimental feature. This feature may be changed or removed without prior notice. Usually there are no bugs. In case you find any bugs, please open an issue in the [TiDB repository](https://github.com/pingcap/tidb/issues). 
```

Reply:
I think the suggestion can be improved further:
```suggestion
> This is an experimental feature. The syntax and implementation may change before GA status.
```
In general I think we should treat released code as production quality, but in this case the syntax is new and an extension of the MySQL dialect, where we want to have some more feedback from community and customers before making the new syntax GA.

Reply:
OK updated.